internal/pkg/mapper/sharedcode/links/utils_test.go

```diff
@@ -1,74 +0,0 @@
-package links
-
-import (
-	"fmt"
-	"testing"
-
-	"github.com/stretchr/testify/assert"
-
-	mapperPkg "github.com/keboola/keboola-as-code/internal/pkg/mapper"
-	"github.com/keboola/keboola-as-code/internal/pkg/model"
-)
-
-func TestSharedCodeLinksMatchId(t *testing.T) {
-	t.Parallel()
-	cases := []struct{ input, expected string }{
-		{input: "", expected: ""},
-		{input: "abc", expected: ""},
-		{input: "{}", expected: ""},
-		{input: "{{}}", expected: ""},
-		{input: "{{{abc}}}", expected: ""},
-		{input: "{{abc/def}}", expected: ""},
-		{input: "{{1}}", expected: "1"},
-		{input: "{{123}}", expected: "123"},
-		{input: "{{a}}", expected: "a"},
-		{input: "{{abc}}", expected: "abc"},
-		{input: "{{A}}", expected: "A"},
-		{input: "{{ABC}}", expected: "ABC"},
-		{input: "  {{ABC}}  \n", expected: "ABC"},
-	}
-	m := NewMapper(nil, mapperPkg.Context{})
-	for i, c := range cases {
-		assert.Equal(t, model.RowId(c.expected), m.matchId(c.input), fmt.Sprintf(`Case "%d"`, i))
-	}
-}
-
-func TestSharedCodeLinksMatchPath(t *testing.T) {
-	t.Parallel()
-	cases := []struct{ input, expected string }{
-		{input: "", expected: ""},
-		{input: "abc", expected: ""},
-		{input: "{}", expected: ""},
-		{input: "{{}}", expected: ""},
-		{input: "{:}", expected: ""},
-		{input: "{{:}}", expected: ""},
-		{input: "{{{:abc}}}", expected: ""},
-		{input: "{{:1}}", expected: "1"},
-		{input: "{{:123}}", expected: "123"},
-		{input: "{{:a}}", expected: "a"},
-		{input: "{{:abc}}", expected: "abc"},
-		{input: "{{:A}}", expected: "A"},
-		{input: "{{:ABC}}", expected: "ABC"},
-		{input: "  {{:ABC}}  \n", expected: "ABC"},
-		{input: "{{:codes/my-code}}", expected: "codes/my-code"},
-		{input: "-- {{:codes/my-code}}", expected: "codes/my-code"}, // SQL comment
-	}
-	m := NewMapper(nil, mapperPkg.Context{})
```

Comment:
Mapper sa v tychto testoch vytvaral bez zavislosti, vid `nil` a prazdny `Context` ^^^.
Po dalsich zmenach to uz nebude mozne.